qa/Rakefile

```diff
@@ -27,21 +27,21 @@ namespace :test do
   namespace :acceptance do
     desc "Run all acceptance"
     task :all do
-      exit(RSpec::Core::Runner.run([Rake::FileList["acceptance/spec/**/*_spec.rb"]]))
+      exit(RSpec::Core::Runner.run([Rake::FileList["acceptance/spec/lib/*_spec.rb"]]))
     end
 
     platforms.types.each do |type|
       desc "Run acceptance test in #{type} machines"
       task type do
-        exit(RSpec::Core::Runner.run([Rake::FileList["acceptance/spec/#{type}/**/*_spec.rb"]]))
+        ENV['LS_TEST_PLATFORM']=type
+        exit(RSpec::Core::Runner.run([Rake::FileList["acceptance/spec/lib/*_spec.rb"]]))
       end
     end
 
     desc "Run one single machine acceptance test"
     task :single, :machine do  |t, args|
```

Comment:
I have been trying to run this command without luck, args is correctly populate with the ubuntu-1204 (also tried with other machines names)

```
~/e/l/qa git:pr/5226 ❯❯❯ bundle exec rake test:acceptance:single\[ubuntu-1204\]                                                                                                                                                                                                                                      ⏎ ✱
No examples found.

Finished in 0.002 seconds (files took 1.3 seconds to load)
0 examples, 0 failures
```